src/Models/ApplicantsReportsModel.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Portal\Models;
+
+use PDO;
+use Portal\Interfaces\ApplicantModelInterface;
```

Comment:
Is this use statement necessary? You do not implement the ApplicantModelInterface

Reply:
Removed it!